ci-operator/step-registry/ipi/conf/alibabacloud/cloud-creds-deprovision/ipi-conf-alibabacloud-cloud-creds-deprovision-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+CR_PATH=${SHARED_DIR}/crs
+
+export ALIBABA_CLOUD_CREDENTIALS_FILE=${SHARED_DIR}/alibabacreds.ini
+cluster_id="${NAMESPACE}-${JOB_NAME_HASH}"
+
+# extract ccoctl from the release image
```

Comment:
We could just use the CCO image directly instead of extracting it from the release image. In other words, modify `ipi-conf-alibabacloud-cloud-creds-deprovision-ref.yaml` to use CCO instead of cli. But the create step would still need `oc`, so perhaps extracting it is better for consistency between the steps.

Reply:
What he has here matches existing `ccotcl` steps, but +1 to pivoting all of these to using the appropriate release image and then [injecting `oc` via `cli: latest`][1].  I'll see about pivoting the existing consumers...

[1]: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli